python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
There is the path to the file, the intent is that you can call get with an argument and request that a file be placed wherever you want as a result of that call. Or in the default, do nothing as the file is available on local storage.